tests/functional/test_install.py

```diff
@@ -270,6 +270,20 @@ def test_install_from_local_directory(script, data):
     assert egg_info_folder in result.files_created, str(result)
 
 
+def test_install_from_local_directory_with_forward_slash(script, data):
+    """
+    Test installing from a local directory.
+    """
+    to_install = data.packages.join("FSPkg")
+    result = script.pip('install', to_install, expect_error=False)
+    fspkg_folder = "/".join([script.site_packages, 'fspkg'])
+    egg_info_folder = "/".join([
+        script.site_packages, 'FSPkg-0.1.dev0-py%s.egg-info' % pyversion
+    ])
+    assert fspkg_folder in result.files_created, str(result.stdout)
+    assert egg_info_folder in result.files_created, str(result)
+
```

Comment:
I'm not clear how this test checks the new functionality. It installs ```data.packages.join("FSPkg")```, which will always use ```os.sep```. Surely the test needs to do something like:

```python
if os.altsep is None:
    return
# Use os.altsep instead of os.sep
to_install = to_install.replace(os.sep, os.altsep)
...
```

Also, we don't run functional tests on Appveyor, so I'd prefer it if this was tested in the unit tests - as the change is a no-op on Unix.

As I'm still not 100% sure I've got the test logic correct above, this is *definitely* something I'd like to see implemented in the form "first implement the test and confirm it fails, then implement the fix and confirm the test now works". If you can do that locally, then that's fine, but if necessary, revert the part of the PR with the actual code change and confirm that the Appveyor run fails on this test before reapplying the code change.